src/NuGet/EtwCollector/Package.nuspec

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Microsoft.ApplicationInsights.EtwCollector</id>
+    <version>$version$</version>
+    <title>Application Insights EtwCollector</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <licenseUrl>http://go.microsoft.com/fwlink/?LinkID=510709</licenseUrl>
+    <projectUrl>http://go.microsoft.com/fwlink/?LinkId=392727</projectUrl>
+    <iconUrl>$image$</iconUrl>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>Application Insights EtwCollector allows sending data from Event Tracing for Windows (ETW) to Application Insights. Application Insights will collect your logs from multiple sources and provide rich powerful search capabilities. Privacy statement: https://go.microsoft.com/fwlink/?LinkId=512156.</description>
+    <summary>Application Insights EtwCollector is supported for .NET Framework 4.5.1 and later.</summary>
+    <copyright>$copyright$</copyright>
+    <releaseNotes>For the release notes please follow http://go.microsoft.com/fwlink/?LinkId=535037</releaseNotes>
+    <tags>Analytics ApplicationInsights Telemetry ASP.NET ASMX Web Azure Server Services ASPX Websites Role Logging Log Tracing ETW EtwCollector Tracing</tags>
+    <dependencies>
+      <group>
+        <dependency id="Microsoft.ApplicationInsights" version="$coresdkversion$" />
+        <dependency id="Microsoft.Diagnostics.Tracing.TraceEvent" version="1.0.41" />
+      </group>
+    </dependencies>
+  </metadata>
+  <files>
+    <!-- Binaries for .NET -->
+    <file src="$configuration$\src\Adapters\Etw.Net451\Microsoft.ApplicationInsights.EtwCollector.dll" target="lib\net451" />
+    <file src="$configuration$\src\Adapters\Etw.Net451\Microsoft.ApplicationInsights.EtwCollector.pdb" target="lib\net451" />
+    <file src="$configuration$\src\Adapters\Etw.Net451\Microsoft.ApplicationInsights.EtwCollector.xml" target="lib\net451" />
+    <!-- XDT transformations -->
+    <file src="$configuration$\src\Nuget\EtwCollector\ApplicationInsights.config.install.xdt" target="content"/>
+    <file src="$configuration$\src\Nuget\EtwCollector\ApplicationInsights.config.uninstall.xdt" target="content"/>
+    <!-- Custom targets -->
+    <file src="$configuration$\src\Nuget\EtwCollector\build\Microsoft.ApplicationInsights.EtwCollector.targets" target="build"/>
```

Comment:
please do not include this target. We have a very bad experience with targets when projects were not compiling after enabling of Application Insights. Or some problems with project reloading happened.